viewCategory.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+include('config/init.php');
+
+$result=getAllCategories();
+
+$categoryPost=getCategoryPost($_GET['categoryId']);
+
+$blogPost=getAllCategoryPost($_GET['categoryId']);
+
+echoHeader($categoryPost['name']);
+
+echo "
+<div class='bordertop'></div>
+<h2>".$categoryPost['name']."</h2>
+";
+
+foreach($blogPost as $blogPosts){
```

Comment:
I think your pluralization is backwards here. The array of posts is singular, and the individual row is plural.